tests/integration/frameworks/test_frameworks.py

```diff
@@ -152,24 +152,8 @@ def test_load(
 ):
     for configuration in test_model.configurations:
         model = framework.load_model(saved_model)
-
         configuration.check_model(model, Resource())
 
-        for method, inps in configuration.test_inputs.items():
-            for inp in inps:
-                args = [inp.preprocess(arg) for arg in inp.input_args]
-                kwargs = {
-                    key: inp.preprocess(kwarg)
-                    for key, kwarg in inp.input_kwargs.items()
-                }
-                if test_model.model_method_caller:
-                    out = test_model.model_method_caller(
-                        test_model, method, args, kwargs
-                    )
-                else:
-                    out = getattr(model, method)(*args, **kwargs)
-                inp.check_output(out)
```

Comment:
Are we removing this because it's calling methods not meant to be called? Not a blocker. Just curious to know.